docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>Add before:</p><p>"Define the processing function that fills the output sample base on the input sample"</p><p> </p><p>Please mention what are the requirements for the function (it must be compatible with @njit, maybe link to it).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>